semanticdb/metadiff/src/main/scala/scala/meta/internal/metadiff/diff/package.scala

```diff
@@ -0,0 +1,84 @@
+package scala.meta.internal.metadiff
+
+import difflib.DiffUtils
+import java.util.regex.Pattern
+import scala.collection.JavaConverters._
+
+package object diff {
```

Comment:
How about moving this to `package object metadiff` instead of introducing a new package?